.github/workflows/ci.yml

```diff
@@ -5,40 +5,42 @@ jobs:
   lint:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/cache@v2
+      - uses: actions/checkout@v3
+      - uses: actions/cache@v3
         with:
           path: '**/node_modules'
           key: ${{ runner.os }}-lint-modules-${{ hashFiles('**/yarn.lock') }}
-      - uses: actions/setup-node@v2
+      - uses: actions/setup-node@v3
         with:
-          node-version: 14.x
-      - run: yarn install
+          node-version: 20
+      - run: yarn install --frozen-lockfile --ignore-scripts
```

Comment:
Let's not do `--ignore-scripts`, it will also ignore scripts from dependencies, which we may need.

Reply:
Oh okay, makes sense. I will remove the additional build step in CI, then, since it would then build the package twice: first when running yarn install (after it has installed, it builds), and a second time in the build step right after.